.github/workflows/pre-commit.yml

```diff
@@ -24,6 +24,9 @@ jobs:
         run: |
           "${GITHUB_WORKSPACE}"/environment_setup/setup_software.sh
       
+      - name: Install pip
+        run: curl -sS https://bootstrap.pypa.io/get-pip.py | python
+
```

Comment:
shouldn't `setup_software` install pip?

Reply:
It should, but the precommit action complains that pip isn't installed. I think it's because the action uses `python` but `python` points to some older installation that maybe doesn't have pip? seems unlikely